pkg/plugins/platform/local.go

```diff
@@ -281,9 +279,34 @@ func (l *LocalApp) DockerComposeYAMLPath() string {
 	return l.AppConfig.DockerComposeYAMLPath()
 }
 
+// ComposeFiles returns a list of compose files for a project.
+func (l *LocalApp) ComposeFiles() []string {
+	files, err := util.FindFiles(path.Join(l.AppRoot(), ".ddev"), "docker-compose")
```

Comment:
I've reworked this to use filepath Glob/Match. I think it's fine to keep the compose files at the top level of .ddev, if we end up having more ddev config files we may want to reconsider this, but I would hope to avoid having multiple ddev config files.